<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="6" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-18T11:18:42 IST" name="Default suite" finished-at="2023-01-18T11:19:01 IST" duration-ms="18614">
    <groups>
    </groups>
    <test started-at="2023-01-18T11:18:42 IST" name="Default test" finished-at="2023-01-18T11:19:01 IST" duration-ms="18614">
      <class name="com.benthonLabs.BenthonLabsApplication.Home_Pages">
        <test-method is-config="true" signature="Browser_Lunch()[pri:0, instance:com.benthonLabs.BenthonLabsApplication.Home_Pages@1fa268de]" started-at="2023-01-18T11:18:42 IST" name="Browser_Lunch" finished-at="2023-01-18T11:18:46 IST" duration-ms="3854" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Browser_Lunch -->
        <test-method signature="HomePage_Open()[pri:1, instance:com.benthonLabs.BenthonLabsApplication.Home_Pages@1fa268de]" started-at="2023-01-18T11:18:46 IST" name="HomePage_Open" finished-at="2023-01-18T11:18:50 IST" duration-ms="4190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HomePage_Open -->
        <test-method signature="EXTENSIVE_EXPERTISE_H2()[pri:6, instance:com.benthonLabs.BenthonLabsApplication.Home_Pages@1fa268de]" started-at="2023-01-18T11:18:50 IST" name="EXTENSIVE_EXPERTISE_H2" finished-at="2023-01-18T11:19:01 IST" duration-ms="10121" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [EXTENSIVE EXPERTISE IN,
TOP-NOTCH PRODUCT DEVELOPMENT SERVICES] but found [Extensive Expertise in, Top-Notch Product Development Services]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [EXTENSIVE EXPERTISE IN,
TOP-NOTCH PRODUCT DEVELOPMENT SERVICES] but found [Extensive Expertise in, Top-Notch Product Development Services]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.benthonLabs.BenthonLabsApplication.Home_Pages.EXTENSIVE_EXPERTISE_H2(Home_Pages.java:201)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EXTENSIVE_EXPERTISE_H2 -->
        <test-method is-config="true" signature="DearDown()[pri:0, instance:com.benthonLabs.BenthonLabsApplication.Home_Pages@1fa268de]" started-at="2023-01-18T11:19:01 IST" name="DearDown" finished-at="2023-01-18T11:19:01 IST" duration-ms="379" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DearDown -->
      </class> <!-- com.benthonLabs.BenthonLabsApplication.Home_Pages -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
